ERROR: Tried to create target "help-glade-da-update-po", but a target of that name already exists.
https://gitlab.gnome.org/GNOME/glade/-/issues/542
https://gitlab.gnome.org/GNOME/glade/-/commit/efdd5338b034a11c5d617684d92d11edc600965e
--- help/LINGUAS.orig
+++ help/LINGUAS
@@ -1,6 +1,5 @@
 # please keep this list sorted alphabetically
 #
-da
 bg
 ca
 cs
